test/metabase/query_processor/pivot_test.clj

```diff
@@ -357,3 +358,49 @@
                   [nil nil 3 607]]
                  (mt/rows
                    (qp.pivot/run-pivot-query query)))))))))
+
+(deftest pivot-with-ids-or-named-fields-in-breakout
+  (mt/dataset sample-dataset
+    (testing "Query works when breakout includes field refs with IDs"
+      (let [query        (mt/mbql-query products
+                           {:breakout [$category
+                                       [:field
+                                        (mt/id :products :created_at)
+                                        {:base-type :type/DateTime :temporal-unit :month}]]
+                            :aggregation [[:count]]})
+            viz-settings (select-keys
+                          (mt/query reviews
```

Comment:
don't understand how this can possibly be expected to work since these columns are from `reviews` and the query is against `products`